.github/actions/deploy-front-end/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Deploy SimpleReport Front End
+description: Build and deploy the React application
+inputs:
+  deploy-env:
+    description: The environment being deployed (e.g. "prod" or "test")
+    required: true
+  smarty-streets-key:
+    description: The Smarty-Streets API token for this environment. (Should be fetched from vault but is not)
+    required: true
+  base-domain-name:
+    description: The domain where the application is deployed (e.g. "simplereport.gov" or "test.simplereport.gov")
+    required: false
+runs:
+  using: composite
+  steps:
+    - name: Install dependencies
+      run: yarn install
+      working-directory: ./frontend
+      shell: bash
+    - name: Set build variables
+      shell: bash
+      run: |
+          az config set extension.use_dynamic_install=yes_without_prompt
+          INSIGHTS_KEY=$(az monitor app-insights component  show -g prime-simple-report-${{inputs.deploy-env}} | jq -r '.[0].instrumentationKey')
```

Comment:
Is the endpoint url also available here (or connection string)?  I've been looking at #1424 and I think we'll need that also so we can send the data to the right place and make sure the url gets into the CSP connect-src.

I don't have permission on azure to try this command, but it looks like it might also be possible to add `--app prime-simple-report-${{inputs.deploy-env}}-insights` and remove the array index.

Reply:
The connection string is also available as a field, but as far as I've been able to tell from manual testing, the key by itself is sufficient for the front end instrumentation. 

I _believe_ that I tried the command you are suggesting and it did not work as expected, but I am not sure--I may have tried something else that didn't work. (I definitely tried several things that didn't work! I will go back and double check if this was one of them.)

Reply:
Answer is no! I had tried something else similar, but not that. Thank you!